

Starting Toontown...
Current time: Tue Dec 22 18:54:44 2015 Pacific Standard Time
sys.path =  ['', 'C:\\Panda3D-1.9.0\\python\\python27.zip', 'C:\\Panda3D-1.9.0\\python\\DLLs', 'C:\\Panda3D-1.9.0\\python\\lib', 'C:\\Panda3D-1.9.0\\python\\lib\\plat-win', 'C:\\Panda3D-1.9.0\\python\\lib\\lib-tk', 'C:\\Panda3D-1.9.0\\python', 'C:\\Panda3D-1.9.0', 'C:\\Panda3D-1.9.0\\bin', 'C:\\Panda3D-1.9.0\\python\\lib\\site-packages']
sys.argv =  ['C:\\Users\\Keithy\\Desktop\\Possible-Toontown-Online-Source-master\\toontown\\toonbase\\ToontownStart.py']
TTLocalizer: Running in language: english
from toontown.toonbase.TTLocalizerEnglish import *
OTPLocalizer: Running in language: english
from otp.otpbase.OTPLocalizerEnglish import *
ToontownStart: Polling for game2 to finish...
ToontownStart: Game2 is finished.
ToontownStart: Starting the game.
ToontownStart: setting default font
ToontownStart: Loading default gui sounds
ToontownStart: serverVersion:  sv-1.0.0
TTAccount: accountServer from launcher:  
TTAccount: default accountServer:  https://toontown.go.com
toontown\safezone\DistributedFindFour.py:598: SyntaxWarning: import * only allowed at module level
  def doRandomMove(self):
Traceback (most recent call last):
  File "C:\Panda3D-1.9.0\python\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Panda3D-1.9.0\python\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\Keithy\Desktop\Possible-Toontown-Online-Source-master\toontown\toonbase\ToontownStart.py", line 88, in <module>
    cr = ToontownClientRepository.ToontownClientRepository(serverVersion, launcher)
  File "toontown\distributed\ToontownClientRepository.py", line 89, in __init__
    prepareAvatar(self.http)
NameError: global name 'prepareAvatar' is not defined
